public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.tsx

```diff
@@ -111,7 +112,12 @@ export const AddAlertRuleModal: FC<AddAlertRuleModalProps> = ({ isVisible, setVi
 
             <TextInputField label={Messages.thresholdField} name="threshold" />
 
-            <NumberInputField label={Messages.durationField} name="duration" validators={[required]} />
+            <NumberInputField
+              inputProps={{ min: MINIMUM_DURATION_VALUE }}
```

Comment:
Wouldn't this produce a native validation error, which is not something we expect to see on UI?

Reply:
Yeah, it would. I added it as a "safeguard", but I believe it won't do anything since the button will be disabled due to the validator. I'm gonna remove it.